pontoon/base/models.py

```diff
@@ -1617,6 +1617,16 @@ def __str__(self):
 
 
 class ProjectLocaleQuerySet(models.QuerySet):
+    def aggregated_stats(self):
```

Comment:
I was wondering if this belongs into a base class, but there seems to be only one other query set that implements this, and differently (huh? on the other one).

So probably not, or at least not as part of this patch.

Reply:
Yeah, we probably want to apply [something like this](https://gist.github.com/mathjazz/a4f0a2023cbb8e8ddebf665cd2325aa7), but I worry that will cause (even more) issues with stats calculation. The reason we use `total_strings` from the `Resource` model in the other method is probably because that value is different from the value in the TranslatedResource when the method is called during sync.